lib/streamlit/web/server/media_file_handler.py

```diff
@@ -62,11 +61,8 @@ def set_extra_headers(self, path: str) -> None:
             filename = media_file.filename
 
             if not filename:
-                title = self.get_argument("title", "", True)
-                title = unquote_plus(title)
-                filename = generate_download_filename_from_title(title)
                 filename = (
-                    f"{filename}{get_extension_for_mimetype(media_file.mimetype)}"
+                    f"StreamlitFile{get_extension_for_mimetype(media_file.mimetype)}"
```

Comment:
nit: could we make this something like `streamlit_file.<file_extension>`? No strong opinion here, but I think that `snake_case` looks better for file names 😆

Reply:
Yes, let's have both, in SiS we will have something like `STREAMLIT_MEDIA_FILE_2E7647FCC300475B99BAFFD59325A743.txt`, I think `MEDIA` part is redundant, so renamed this to `STREAMLIT_MEDIA.extension`